src/resources/parser/glb-parser.js

```diff
@@ -2293,7 +2294,9 @@ class GlbParser {
     constructor(device, assets, maxRetries) {
         this._device = device;
         this._assets = assets;
-        this._defaultMaterial = getDefaultMaterial(device);
+        this._defaultMaterial = createMaterial({
```

Comment:
What is the difference between the material created here and the device default material? It looks like the device default material has had [shadingModel](https://github.com/playcanvas/engine/blob/main/src/framework/app-base.js#L648) set, but that's the standard material [default anyway](https://github.com/playcanvas/engine/blob/main/src/scene/materials/standard-material.js#L1057).

Is it missing:
```
    material.diffuseTint = true;
    material.diffuseVertexColor = true;

    material.specularTint = true;
    material.specularVertexColor = true;
```
?

Reply:
yes, those options are what make the vertex colors to work

Reply:
So why aren't you setting the options on the material here? What am I missing? :)

Reply:
I don't want to modify the global default material - that one is used by many other things. So we create a glb specific material here. 
I though about having a single global one .. but creating one per glb is probably a good idea, so that the use can modify it further without affecting other glbs

Reply:
Sorry, what I mean is: you're not setting 'diffuseTint' or 'diffuseVertexColor' on this material?

Reply:
the createMaterial function does that, which I'm calling.

Reply:
Yes, right that's what I missed. `Face palm`. Thanks for your patience at this time!